04_environment/index.tex

```diff
@@ -1 +1,283 @@
+    \centering
+    \includegraphics[width=1\textwidth]{04_environment/Images/Distribution of Foraging returns Deer Hunting.PNG}
+    \caption{Deer hunting payoff dominant Distribution for foraging return}
+    \label{fig:Distribution of Foraging returns Deer Hunting}
+\end{figure}
+
+To increase the risk, the Bernoulli Random Variable ($D$) has been utilised to not guarantee any return resources after a hunt. Whilst the Exponential Decay ($W$) is to disincentive too much resources being placed in hunting and hoping for a high return. However this all comes with the benefit of significantly higher ($2x$) returns than fishing.
+
+Figure~\ref{fig:Distribution of Foraging returns Deer Hunting} represents the return utility which represents the return multiplier of the foraging. The average \textbf{expected mean} return across $1000$ iterations is shown by the colourful lines which is close to the \textbf{real mean}.
+
+\newpage
+\subsubsection{Fish hunting - Risk Aversion}
+
+Fishing is similar to Deer hunting, except that it uses only a Normal Distribution ($F$), which focuses on avoiding risk by being very predictable and safe. Figure~\ref{fig:Distribution of Foraging returns Fishing} illustrates the return utility function of fishing.
+
+\begin{figure}[!htb]
+    \centering
+    \includegraphics[width=1\textwidth]{04_environment/Images/Distribution of Foraging returns Fishing.PNG}
+    \caption{Fishing Risk aversion Distribution for foraging return}
+    \label{fig:Distribution of Foraging returns Fishing}
+\end{figure}
+
+As seen in Figure~\ref{fig:Distribution of Foraging returns Deer Hunting}and Figure~\ref{fig:Distribution of Foraging returns Fishing}, the return for the deer is almost double that of the fish return whilst the fish return is much easier to obtain due to the lower cost of catching fish.
+
+\newpage
+\subsection{Solution Concepts}
+\subsubsection{Dominant Strategy}
+
+In this implementation, there is no dominant strategy. Therefore, if an agent decides to go deer hunting, whereas all the other islands are fishing that agent would not get the best return possible. Hence it is not the best strategy. Whilst if an agent goes fishing and there are a sufficient amount of agents deer hunting, then that agent would be passing on the opportunity of gaining a larger return in deer hunting. Hence, it is not the best strategy either.
+
+\subsubsection{Nash Equilibrium}
+
+In the regular stag hunt game there are two pure Nash equilibria, where both players go for either payoff or risk aversion. In our implementation there are multiple Nash Equilibria; these would be the points where the islands cannot benefit themselves by moving to deers hunting to generate more return. As there is a limit to how many deers that can be hunted thus by moving to deer hunting they are getting a worse return than if they just stayed at fishing. Whilst the islands at the deer hunt are already making a greater return and have no reason to switch to fishing.
```

Comment:
In the regular stag hunt game,

Reply:
**In our implementation there are multiple Nash Equilibria.**  These **are** the points...